neurom/analysis/morphtree.py

```diff
@@ -237,11 +237,11 @@ def get_bounding_box(tree):
     [xmax, ymax, zmax]]
     """
 
-    min_x = min(p[0] for p in val_iter(tr.ipreorder(tree)))
-    min_y = min(p[1] for p in val_iter(tr.ipreorder(tree)))
-    min_z = min(p[2] for p in val_iter(tr.ipreorder(tree)))
-    max_x = max(p[0] for p in val_iter(tr.ipreorder(tree)))
-    max_y = max(p[1] for p in val_iter(tr.ipreorder(tree)))
-    max_z = max(p[2] for p in val_iter(tr.ipreorder(tree)))
+    min_x = min(p[COLS.X] for p in val_iter(tr.ipreorder(tree)))
```

Comment:
This seems inefficient, couldn't you do something like:
min_value, max_value = np.array([np.INF, np.INF, np.INF), np.array([np.NINF, np.NINF, np.NINF)
for p in val_iter(tr.ipreorder(tree)):
    min_value = np.minimum(p, min_value)
    max_value = np.maximum(p, max_value)
return min_value, max_value

That what it's one traversal instead of 6